frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -61,7 +61,23 @@ export const saveDashboardAndCards = createThunkAction(
           ),
       );
 
-      // add isAdded dashboards
+      // update parameter mappings
+      dashboard.ordered_cards = dashboard.ordered_cards.map(dc => ({
```

Comment:
This was pulled up from [here](https://github.com/metabase/metabase/pull/27708/files#diff-15053bd90a3e6346e07e53504670ead95ee0bb822a85f7bc00a55f9ec324cc2bL131). We need to update the parameter_mappings before adding the cards to the dashboard now.